voeding.html

```diff
@@ -0,0 +1,113 @@
+        <script type="text/javascript" src="js/script.js"></script>
+    </head>
+    <body>
+        <div id="wrapper">
+            <nav id="menu">
+                <div class="menu-item logo">
+                    <a href="index.html" title="Home">
+                        <img src="images/logo.png" alt="Logo" width="50">
+                    </a>
+                </div>
+                <div class="menu-item title"><a href="index.html" title="home"><span class="brand-primary">Bewust</span> <span class="brand-secondary">Kopen</span></a></div>
+                <div id="nav-buttons">
+                    <div class="menu-item menu-button">
+                        <a href="index.html" title="Home">Home</a>
+                    </div>
+                    <div class="menu-item menu-button">
+                        <a class="active" href="app.html" title="Applicatie">Applicatie</a>
+                    </div>
+                    <div class="menu-item menu-button">
+                        <a href="about.html" title="Over ons">Over Ons</a>
+                    </div>
+                    <div class="menu-item menu-button">
+                        <a href="voeding.html" title="Voeding">Voeding</a>
+                    </div>
+                    <div class="menu-item menu-button">
+                        <a href="sponsor.html" title="Sponsoren">Sponsoren</a>
+                    </div>
+                    <div class="menu-item menu-button">
+                        <a href="contact.html" title="Contact">Contact</a>
+                    </div>
+                    <div class="menu-item search-box">
+
+                        <input type="text" name="search" style="height: 60%; margin: 0 5px;" placeholder="Zoeken...">
+                    </div>
+                    <div class="social-wrapper">
+                        <a class="social-icon" href="https://www.facebook.com/bewustkopen">
+                            <img src="images/facebook.png" alt="Facebook" width="25" height="25">
+                        </a>
+                        <a class="social-icon" href="https://www.twtter.com/bewustkopen">
+                            <img src="images/twitter.png" alt="Twitter" width="25" height="25">
+                        </a>
+                        <a class="social-icon" href="https://www.pinterest.com/bewustkopen">
+                            <img src="images/pinterest.png" alt="Pinterest" width="25" height="25">
+                        </a>
+                    </div>
+                </div>
+                <div class="menu-button-mobile"></div>
+            </nav>
+            <div id="title-bar">
+                <h1 class="title">Voeding</h1>
+                <div class="breadcrumb"><a href="index.html" title="home">Home</a>-&gt;<a href="voeding.html" title="voeding">Voeding</a>
+            <div id="content">
+                <img style="float:right; width:300px;" src="images/vegetables.jpg">
```

Comment:
Zorg er voor dat de vormgeving in het CSS bestand wordt gedaan. Voor de afmetingen van een afbeelding kun je beter de attributen width en height gebruiken. Bijvoorbeeld: `<img src="plaatje.jpg" width="100" height="200">`.